web/src/settings_users.js

```diff
@@ -380,7 +380,7 @@ export function update_bot_data(bot_user_id) {
         return;
     }
 
-    bot_list_widget.render_item(bot_user_id);
+    bot_list_widget.render_item(bot_info(bot_user_id));
```

Comment:
What's the story with this change? It seems like a functional change unrelated to the typescript adjustment.